src/content_panel/panel.macros.html

```diff
@@ -5,10 +5,10 @@
 {#
   * Renders a content and image panel.
 #}
-{% macro content_panel(heading, image_path, image_description=null, image_position='left', class=null, id=null) %}
+{% macro content_panel(heading, image_path, image_description=null, image_position='left', imgix_class=null, class=null, id=null) %}
```

Comment:
I dig `imgix_class` as the standard param key for any module that has media coming from imgix.